Packs/AWS-IAM/Integrations/AWS-IAM/AWS-IAM.py

```diff
@@ -385,6 +387,38 @@ def delete_access_key(args, client):  # pragma: no cover
         demisto.results("The Access Key was deleted")
 
 
+def list_mfa_devices(args, client):
+    response = client.list_mfa_devices(UserName=args.get('userName'))
+    mfa_devices = response['MFADevices']
+    data = []
+    for mfa_device in mfa_devices:
+        data.append({
+            'UserName': mfa_device['UserName'],
+            'SerialNumber': mfa_device['SerialNumber'],
+            'EnableDate': datetime.strftime(mfa_device['EnableDate'], '%Y-%m-%d %H:%M:%S'),
+        })
+        ec = {'AWS.IAM.MFADevices': data}
+        human_readable = tableToMarkdown('AWS IAM Users MFA Devices', data)
+        return_outputs(human_readable, ec)
+
+
+def deactivate_mfa_device(args, client):
+    response = client.deactivate_mfa_device(
+        UserName=args.get('userName'),
+        SerialNumber=args.get('serialNumber')
```

Comment:
```suggestion
        UserName=args['userName'],
        SerialNumber=args['serialNumber']
```
these are required args